src/lib/middleware/redisPageCache.ts

```diff
@@ -55,6 +56,12 @@ export function redisPageCacheMiddleware(
     return next()
   }
 
+  // TODO: remove once "artist_grid_manual_curation_trial" A/B test is over
+  // Skip targeted artist routes
+  if (TARGETED_ARTIST_ROUTES.includes(req._parsedUrl.pathname)) {
+    return next()
+  }
+
```

Comment:
There was a problem with A/B testing (see [this thread](https://artsy.slack.com/archives/C9SATFLUU/p1647011695022429) for more details) due to page caching. For this reason, caching has been disabled for the target artist pages.

@damassi @mzikherman  when you have a free minute, please take a look if everything is OK